roles/openshift_node/tasks/upgrade_pre.yml

```diff
@@ -5,6 +5,18 @@
 - set_fact:
     skip_node_svc_handlers: True
 
+- name: Wait for node to be ready
```

Comment:
Why are we waiting here?  This is the pre_upgrade step.  Nothing should have been restarted at this point.

Reply:
An unready node can’t be safely upgraded when bootstrapping is in play.  I think if this as a precondition rather than a failure mode.

Reply:
Will verify this is still needed.